<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="bookTab">
    <attribute name="title">
     <string>Book</string>
    </attribute>
    <widget class="QLineEdit" name="bookSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="bookTable">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addBookButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Book</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeBookButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Old Book</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="readerTab">
    <attribute name="title">
     <string>Reader</string>
    </attribute>
    <widget class="QLineEdit" name="readerSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="readerTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addReaderButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Reader</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeReaderButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Reader</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyReaderButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify Reader</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
